Source/UZKArchive.m

```diff
@@ -836,7 +840,7 @@ - (BOOL)extractBufferedDataFromFile:(NSString *)filePath
     NSProgress *progress = [self beginProgressOperation:0];
     
     __weak UZKArchive *welf = self;
-    const NSUInteger bufferSize = 4096; //Arbitrary
+    NSUInteger bufferSize = 4096 * 64; //Arbitrary
```

Comment:
@amosavian What was the specific benefit of this change, and how did you arrive at 64 as the multiple (and not 32, 16, etc.)? Won't it just cause less frequent updates to the caller?

Reply:
That was based on my app performance on updating UI's progress bar in my iOS application to make it as smooth as possible. Nowadays processors are fast enough to process 256kb in few milliseconds. Values less than this probably makes the whole decompressing process slow.

 Maybe a better choice is allocating buffer based on total file size.

Reply:
I'm thinking of providing an overload to allow the caller to choose the buffer size, but still provide this overload with the defaulted value. Would that work for you, and can you see any downsides?

Reply:
That would lead to more confusion. It’s not easy to spot best chunk size without try and error by developer.
Anyway, Reporting progress in small intervals puts huge unnecessary overload on cpu. I think we must use no less than 256kb as chunk size, at least for a real scenario.

Reply:
I agree that 256 kb is a better default, but I think I would like to add the ability to change that in a future release. Take a look at the latest diffs and let me know if everything looks good. If so, I'll merge and cut a new beta.